docs/ensnode.io/src/content/docs/ensnode/concepts/what-is-ensnode.mdx

```diff
@@ -16,6 +18,12 @@ ENSNode is the new multichain indexer for ENS and ENSv2. It is built on [Ponder]
 
 ENSNode version 1 (`V1`), discussed here, prioritizes equivalency with the [ENS Subgraph](/ensnode/understanding/the-ens-subgraph/), which drove many architectural decisions.
 
+<LinkCard
+  title="ENSNode Roadmap"
+  description="Learn about ENSNode's future roadmap."
+  href="/ensnode/roadmap/"
```

Comment:
This is 404 at the moment:
https://ensnode-jwrv6wjnp-namehash.vercel.app/ensnode/roadmap/

A working link right now is:
https://ensnode-jwrv6wjnp-namehash.vercel.app/ensnode/concepts/roadmap

We need to update one of them, I suppose.